api/src/main/resources/public/api/v1/examples/queue_job_list.json

```diff
@@ -0,0 +1,48 @@
+[
+  {
+    "job":"fatjob2",
```

Comment:
shouldn't this be "id" not "job"? or "jobId"...

We use the word id for job id on most places as I have seen